src/common/components/Rarenessbar/Rarenessbar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import cn from 'classnames'
+import Tooltip from '../Tooltip'
+
+export type TRarenessProps = {
+  percent: number
+  classes?: string
+}
+
+const Rarenessbar: React.FC<TRarenessProps> = ({
```

Comment:
Not a big deal, but my editor checks for grammar errors, and such variables will be highlighted. Please use the upper letter to separate words: RarenessBar